Documentation/Metrics/arangodb_scheduler_jobs_dequeued.yaml

```diff
@@ -9,5 +9,15 @@ exposedBy:
   - coordinator
   - dbserver
   - agent
+  - single
 description: |
-  Total number of jobs dequeued.
+  The total number of jobs dequeued from all scheduler queues.
```

Comment:
I would have suggested to not document the legacy metrics at all, but I have no strong feelings. I would have suggested to remove this YAML file and use `DECLARE_LEGACY_GAUGE` such that the check script does not complain. If we leave it like this we have the strange situation that the manual has a duplication of the documentation but the v2 metrics do not have some of the entries.

Reply:
Makes sense!